server/repository/src/migrations/v2_10_00/rename_cold_storage_type_to_location_type.rs

```diff
@@ -17,7 +17,6 @@ impl MigrationFragment for Migrate {
                     ALTER TABLE item_variant RENAME COLUMN cold_storage_type_id TO location_type_id;
                 "#
         )?;
-        // if cfg!(feature = "postgres") {}
```

Comment:
Remove?

Reply:
yeah was leftover comment here by the looks?